### PR TITLE
Using flask_admin instead of deprecated flask.ext.admin

### PR DESCRIPTION
**Pull request by amilosmanli**

### CODE REVIEW

File: flask_admin_s3_upload.py [hunk 1]
```diff
@@ -18,10 +18,10 @@
 from werkzeug.datastructures import FileStorage
 
 from wtforms import ValidationError
-
-from flask.ext.admin.form.upload import FileUploadField, ImageUploadInput, \
+x
```
Comment (Jaza) [inline flask_admin_s3_upload.py]: Stray 'x' here, please remove

**Comment by Jaza** [comment]
Thanks @cyberlake !!

**Comment by amilosmanli** [comment]
@Jaza can you push this version to pypi please?

**Comment by Jaza** [comment]
@cyberlake - ok, done, released v0.1.4 and pushed to PyPI. Hope that helps.

**Pull request merged**